tools/python/boutiques/neurolinks-template/tool.owl

```diff
@@ -0,0 +1,7 @@
+nl:neurodocker rdf:type owl:ObjectProperty ;
```

Comment:
`nl:neurodocker` was specific for a certain tool - we want to set a unique identifier of `nl:.....` for each new tool, and ensure that it is unique